avocado/utils/genio.py

```diff
@@ -203,9 +203,10 @@ def write_file_or_fail(filename, data):
                          filename, details))
 
 
-def check_pattern_in_file(filename,  pattern):
+def is_pattern_in_file(filename,  pattern):
```

Comment:
This rename will also need to be done in the tests.  BTW, feel free to squash those changes... there's usually no reason to keep the result of review interactions on the project history.  You can squash *your* changes, and leave the original author unchanged, or even merge them, whatever you prefer.

Reply:
Too hurry. I'm going to open a proper v3 squashing my changes and fixing it.